### PR TITLE
NEST-380: Clarify documentation for theming with Media Theme

### LINKED ISSUE #16: Documentation mistakes (NEST-381)
- The solution file copy in your fork doesn't seem to be necessary. It's better if you just rename the solution, because then changes made in the official repo can be easily merged too.
- "From here on, general Orchard Core theme development rules apply with some DotNest-related extras. You can read about all these on the Theming a DotNest site page of the DotNest Knowledge Base." has two issues:
  - It doesn't explain nor point to what those "DotNest-related extras" are.
  - The link points to an Orchard 1 DotNest knowledge base page.

[Jira issue](https://lombiq.atlassian.net/browse/NEST-381)

### LINKED ISSUE #45: Clarify documentation for theming with Media Theme (NEST-380)
Currently, https://dotnest.com/knowledge-base/topics/theming-orchard-core points you to the Readme of [Media Theme](https://github.com/Lombiq/Hosting-Media-Theme), that explains a lot of things that are interesting for when you use it in your own DotNest-like project. Some of it is indeed interesting for working with your DotNest site, but part of it is under https://github.com/Lombiq/DotNest-Core-SDK (what you have to use as well).

- Clarify the whole story in a way that when you want to theme your DotNest site, then you have a single place to understand what to do, step by step, so you don't have to match it together from the two Readmes (also keeping in mind what doesn't apply to you).
- The SDK Readme explains in great detail how to set up sync with Git-hg Mirror. However, this is a minor and optional recommendation, so should be kept as such, after the main points of the project. Also, add there something along the lines of:

```md
Note that this project uses [GitHub Actions](https://github.com/features/actions) for automated builds. If you don't need these, and especially if your fork will be auto-updated with Git-hg Mirror (which pushes to your repository multiple times a minute) then you shouldn't enable GitHub Actions for the repository (since then e.g. a build will run on each Git-hg Mirror push).
```

[Jira issue](https://lombiq.atlassian.net/browse/NEST-380)

### PR DESCRIPTION
**Pull request by Piedone**
[NEST-380](https://lombiq.atlassian.net/browse/NEST-380)
Fixes https://github.com/Lombiq/DotNest-Support/issues/45.
Fixes #16.

### CODE REVIEW

File: Readme.md [hunk 1]
```diff
@@ -31,16 +41,17 @@ For example, if the project you're working is called `Awesome Project`, then you
 
 ## Theme development
 
-- Create a custom solution file on your prefixed development branch as a copy of `DotNest.Core.SDK.sln`, e.g. `AwesomeProject.sln`.
-- From here on, general Orchard Core theme development rules apply with some DotNest-related extras. You can read about all these on the [`Theming a DotNest site` page of the DotNest Knowledge Base](https://dotnest.com/knowledge-base/topics/theming/).
-- If your theme contains Liquid templates, enable the `Liquid` feature for these to be picked up by Orchard Core.
-- The Media Theme on DotNest also has an automated mechanism to include some site-level resources. This might come in handy e.g. if your theme doesn't have a base theme and/or you're not overriding the `Document` or `Layout` shapes.
```
Comment (Piedone) [inline Readme.md:37]: I don't see any indication of this actually being implemented, but let me know if it actually is.

Comment (wAsnk) [reply Readme.md:37]: Me neither. @barthamark did you implement such feature?

Comment (wAsnk) [reply Readme.md:37]: Or @porgabi , you added this line. Is this just a copy paste error from [https://github.com/Lombiq/DotNest-SDK](https://github.com/Lombiq/DotNest-SDK) ?

Comment (wAsnk) [reply Readme.md:37]: OK, this is clearly a copy paste error.

**Pull request merged**